app/modules.go

```diff
@@ -220,37 +170,49 @@ func orderBeginBlockers() []string {
 	}
 }
 
+func OrderEndBlockers(allModuleNames []string) []string {
+	ord := partialord.NewPartialOrdering(allModuleNames)
+	// Epochs must run after all other end blocks
+	ord.LastElements(epochstypes.ModuleName)
+	// txfees auto-swap code should occur before any potential gamm end block code.
+	ord.Before(txfeestypes.ModuleName, gammtypes.ModuleName)
+	// only remaining modules that aren;t no-ops are: crisis & govtypes
+	// we don't care about the relative ordering between them.
+
+	return ord.TotalOrdering()
+}
+
 // orderEndBlockers Tell the app's module manager how to set the order of
 // EndBlockers, which are run at the end of every block.
 var orderEndBlockers = []string{
-	lockuptypes.ModuleName,
+	lockuptypes.ModuleName, // no-op
 	crisistypes.ModuleName,
 	govtypes.ModuleName,
-	stakingtypes.ModuleName,
-	capabilitytypes.ModuleName,
-	authtypes.ModuleName,
-	banktypes.ModuleName,
-	distrtypes.ModuleName,
-	slashingtypes.ModuleName,
-	minttypes.ModuleName,
-	genutiltypes.ModuleName,
-	evidencetypes.ModuleName,
-	authz.ModuleName,
-	paramstypes.ModuleName,
-	upgradetypes.ModuleName,
-	vestingtypes.ModuleName,
-	ibchost.ModuleName,
-	ibctransfertypes.ModuleName,
-	gammtypes.ModuleName,
-	incentivestypes.ModuleName,
-	lockuptypes.ModuleName,
-	poolincentivestypes.ModuleName,
-	superfluidtypes.ModuleName,
-	bech32ibctypes.ModuleName,
-	txfeestypes.ModuleName,
+	stakingtypes.ModuleName,        // no-op
+	capabilitytypes.ModuleName,     // no-op
+	authtypes.ModuleName,           // no-op
+	banktypes.ModuleName,           // no-op
+	distrtypes.ModuleName,          // no-op
+	slashingtypes.ModuleName,       // no-op
+	minttypes.ModuleName,           // no-op
+	genutiltypes.ModuleName,        // (can't tell if no-op)
```

Comment:
Did you confirm that this one is a no-op? Just making sure it wasn't missed

Reply:
Just found it: https://github.com/cosmos/cosmos-sdk/blob/v0.45.4/x/genutil/module.go#L85

I missed finding it on my first pass, because I missed that line